### PR TITLE
Add DisableUpdates & clean up

### PR DESCRIPTION
**Pull request by calvinpark**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,17 +139,21 @@ You'll need a C3 or C3X and a Toyota A harness for current supported vehicles.
 
 ### Step 1. Power up C3X, connect to Wi-Fi, and install `commaai/master-ci`
 
-1-1. Start with the official installation guide: https://comma.ai/setup/comma-3x
+1-1. Install the hardware by following the official Installation Guide: https://comma.ai/setup/comma-3x
```
Comment (nelsonjchen) [inline README.md:142]: I wonder if it's worth linking to installation videos for Toyotas in general.

File: README.md [hunk 2]
```diff
@@ -139,17 +139,21 @@ You'll need a C3 or C3X and a Toyota A harness for current supported vehicles.
...
 
-1-2. Always use the right-angled USB-C cable that came with the Comma 3X. Don't use your own USB cable. A USB-C 3.1 Gen 2 is required.
+1-2. Turn on your Comma 3X by plugging in the right-angled OBD-C cable to the harness.
 
-1-3. Also connect the OBD2 connector to make sure the Comma stays powered on while turning the car on and off.
+> [!WARNING]
+> * For connecting Comma to the car harness, always use the right-angled OBD-C cable that came with the Comma 3X. The hardness sends 12V instead of the usual 5V, so your own USB cable will not work.
```
Comment (nelsonjchen) [inline README.md:147]: case thing as mentioned. Also harness not hardness. 

The voltage is not related things working or not. However, maybe it should be noted that it is *not* USB and that plugging things in that aren't C3s may cause damage.

File: README.md [hunk 3]
```diff
@@ -139,17 +139,21 @@ You'll need a C3 or C3X and a Toyota A harness for current supported vehicles.
...
+> * Comma AI sells it if you need more: https://comma.ai/shop/obd-c-cable
+> * If you must buy your own, USB-C 3.1 Gen 2 is required.  
+> * For turning on Comma at home, your phone charger works fine. It will show a voltage warning but it will work.
```
Comment (nelsonjchen) [inline README.md:150]: This should be noted as a YMMV. The C3 doesn't really follow standards and so far, the most success has been USB A-C but C-C sometimes works too.

File: README.md [hunk 4]
```diff
@@ -139,17 +139,21 @@ You'll need a C3 or C3X and a Toyota A harness for current supported vehicles.
 
 ### Step 1. Power up C3X, connect to Wi-Fi, and install `commaai/master-ci`
```
Comment (nelsonjchen) [inline README.md:140]: I wonder if we should just refer everything as a C3 and just make a note here that this applies to the newer C3X as well. Sometimes we do get people asking if a C3 would work.

File: README.md (hunk 1 above)
Comment (calvinpark) [reply README.md:142]: Good idea

File: README.md (hunk 2 above)
Comment (calvinpark) [reply README.md:147]: Is it the speed that's needed?

File: README.md (hunk 4 above)
Comment (calvinpark) [reply README.md:140]: That could work. I was thinking that the newbies who need this have C3X

File: README.md (hunk 1 above)
Comment (calvinpark) [reply README.md:142]: The most comprehensive one is this but it talks about comma power and ethernet. May need a new one. https://www.youtube.com/watch?v=_VWCt4UMXUY

File: README.md (hunk 4 above)
Comment (calvinpark) [reply README.md:140]: Mind if I do this on a different PR? I think we should get the DisableUpdates change pushed out soon

File: README.md (hunk 1 above)
Comment (calvinpark) [reply README.md:142]: Same, maybe on a later PR?

File: README.md (hunk 2 above)
Comment (nelsonjchen) [reply README.md]: It's just all pins that's needed.

**Pull request merged**